### PR TITLE
Modify  by adding <div> to htmls

### PR DESCRIPTION
**Pull request by LiXiang618**

### CODE REVIEW

File: 02 Algorithm Reference/08 Scheduled Events/02 DateTime Rules/content.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
 				</td>
 				         
 				<td>
-					<pre class="prettyprint">DateRules.EveryDay("SPY")</pre>
+					<code class="prettyprint">DateRules.EveryDay("SPY")</code>
```
Comment (gaviles) [inline 02 Algorithm Reference/08 Scheduled Events/02 DateTime Rules/content.html]: @LiXiang618  please also remove `class="prettyprint" `

File: 02 Algorithm Reference/11 Universes/02 Basic Universe API/content.html [hunk 2]
```diff
@@ -8,14 +8,17 @@
 	<p>
 		<b>Universe Settings</b>
 		<br/>
-			If you do not pass in a full universe object the $[UniverseSettings] property is used to fill in the blanks.  Changing the UniverseSettings algorithm property can be helpful to simplify adding universes. Universes have 4 key properties:<pre>
+			If you do not pass in a full universe object the $[UniverseSettings] property is used to fill in the blanks.  Changing the UniverseSettings algorithm property can be helpful to simplify adding universes. Universes have 4 key properties:
+			<div class="section-example-container">	
+			<pre>
```
Comment (gaviles) [inline 02 Algorithm Reference/11 Universes/02 Basic Universe API/content.html]: this pre is missing the class 'all'

File: 02 Algorithm Reference/11 Universes/03 Coarse Universe Selection/content.html [hunk 3]
```diff
@@ -4,13 +4,16 @@
 	Coarse Universe selection is the built in universe data provided by QuantConnect. Using financial data we generate a few key properties for each symbol and allow you to filter the universe of 16,400+ symbols to receive the symbols matching your filter criteria. It uses the $[CoarseFundamental] type.
 </p>
 <p>
-	Coarse fundamental has the following properties you can use to perform rough filtering.<pre>class CoarseFundamental {
+	Coarse fundamental has the following properties you can use to perform rough filtering.
+	<div class="section-example-container">	
+	<pre>class CoarseFundamental {
```
Comment (gaviles) [inline 02 Algorithm Reference/11 Universes/03 Coarse Universe Selection/content.html]: this pre is missing the class 'all'

File: 02 Algorithm Reference/13 Reality Modelling/01 Introduction to Models/content.html [hunk 4]
```diff
@@ -6,8 +6,11 @@
 	Models can be used to improve the accuracy of your backtesting. We provide basic default models which assume you are trading on highly liquid assets, but if you are trading high volumes, or on low volume assets you should update these models to be more realistic.
 </p>
 <p>
-	All models are set on a per security basis. To set a model first fetch the security object and apply your model. <pre>//Set IBM to have a constant $1 transaction fee. 
+	All models are set on a per security basis. To set a model first fetch the security object and apply your model. 
+	<div class="section-example-container">	
+	<pre>//Set IBM to have a constant $1 transaction fee.
```
Comment (gaviles) [inline 02 Algorithm Reference/13 Reality Modelling/01 Introduction to Models/content.html]: missing class 'all'

File: 02 Algorithm Reference/13 Reality Modelling/05 Fill Models/content.html [hunk 5]
```diff
@@ -6,14 +6,18 @@
 	Fill models give you control over order fills. Each supported order type is passed through a dedicated method and returns an $[OrderEvent] object. OrderEvents are used to carry information about order partial fills or errors.
 </p>
 <p>
-	The Fill Models implement the $[IFillModel] interface. They have the following key methods:<pre>public interface IFillModel {
+	The Fill Models implement the $[IFillModel] interface. They have the following key methods:
+	<div class="section-example-container">	
+	<pre>public interface IFillModel {
```
Comment (gaviles) [inline 02 Algorithm Reference/13 Reality Modelling/05 Fill Models/content.html]: missing class all

File: 02 Algorithm Reference/14 Charting/03 Supported Series Types/content.html [hunk 6]
```diff
@@ -2,11 +2,14 @@
 <meta name="tag" content="adding charts" />
 <meta name="tag" content="series types" />
 <p>
-	The charting API supports the follow series types. Nothing special is required to use these series, simply specify them for your series when creating your chart.<pre>
+	The charting API supports the follow series types. Nothing special is required to use these series, simply specify them for your series when creating your chart.
+	<div class="section-example-container">
+	<pre>
```
Comment (gaviles) [inline 02 Algorithm Reference/14 Charting/03 Supported Series Types/content.html]: missing class all

File: 02 Algorithm Reference/14 Charting/04 Custom Colors and Scatter Symbols/content.html [hunk 7]
```diff
@@ -8,11 +8,14 @@
 	the parameter <code>"Color.ColorName"</code>.
 </p>
 <p>
-	You could also change the marker symbol of the scatter plot. The charting API supports the following scatter marker symbol types.<pre>
+	You could also change the marker symbol of the scatter plot. The charting API supports the following scatter marker symbol types.
+	<div class="section-example-container">
+	<pre>
```
Comment (gaviles) [inline 02 Algorithm Reference/14 Charting/04 Custom Colors and Scatter Symbols/content.html]: missing class all

File: 03 Asset Classes/01 Equities/01 Equities/content.html [hunk 8]
```diff
@@ -34,6 +34,7 @@ <h3>
 	</h3>
 	Subscribe to equity data:
 	<p>
+		<div class="section-example-container">
 		<pre>
```
Comment (gaviles) [inline 03 Asset Classes/01 Equities/01 Equities/content.html]: missing class all

File: 05 Research/01 Introduction to Jupyter/01 Introduction/content.html [hunk 9]
```diff
@@ -13,6 +13,7 @@
 <p>
 	The foundational class for access QuantConnect data in Jupyter is the <code>QuantBook</code>. This is a wrapper on the QCAlgorithm class and has many of the same methods. The research environment is currently in beta but we are slowly adding new methods to the QuantBook to expand your research potential. The basic template provides the foundation code required to import the QuantBook class. Specifically: 
 </p>
+<div class="section-example-container">
 <pre class="prettyprint">
```
Comment (gaviles) [inline 05 Research/01 Introduction to Jupyter/01 Introduction/content.html]: please remove prettyprint class and add all

File: 02 Algorithm Reference/13 Reality Modelling/05 Fill Models/content.html [hunk 10]
```diff
@@ -8,21 +8,21 @@
 <p>
 	The Fill Models implement the $[IFillModel] interface. They have the following key methods:
 	<div class="section-example-container">	
-	<pre>public interface IFillModel {
+	<<pre class = "all">>public interface IFillModel {
```
Comment (gaviles) [inline 02 Algorithm Reference/13 Reality Modelling/05 Fill Models/content.html]: @LiXiang618  this is not correct

File: 02 Algorithm Reference/13 Reality Modelling/05 Fill Models/content.html [hunk 11]
```diff
@@ -8,21 +8,21 @@
...
     OrderEvent MarketFill(Security asset, MarketOrder order);
     OrderEvent StopMarketFill(Security asset, StopMarketOrder order);
     OrderEvent StopLimitFill(Security asset, StopLimitOrder order);
     OrderEvent LimitFill(Security asset, LimitOrder order);
     OrderEvent MarketOnOpenFill(Security asset, MarketOnOpenOrder order);
     OrderEvent MarketOnCloseFill(Security asset, MarketOnCloseOrder order);
-}</pre>
+}</<pre class = "all">>
```
Comment (gaviles) [inline 02 Algorithm Reference/13 Reality Modelling/05 Fill Models/content.html]: same here

File: 03 Asset Classes/01 Equities/01 Equities/content.html [hunk 12]
```diff
@@ -13,10 +13,10 @@ <h5 style="font-weight: bold;">
 		Our data is provided by QuantQuote and is <b>trade</b> data only only with tick, second, minute, hour and daily resolutions available.
 	</p>
 	<p>
-		The Tick data is raw and unfiltered - which includes suspicious, untradable and off market ticks. This can often result in unrealistic backtests. Before using tick data you should fully understand all the exchange codes and the "suspicious" flag which indicates a tick you should ignore. Unless you're an expert you should use the pre-filtered second and minute data.
+		The Tick data is raw and unfiltered - which includes suspicious, untradable and off market ticks. This can often result in unrealistic backtests. Before using tick data you should fully understand all the exchange codes and the "suspicious" flag which indicates a tick you should ignore. Unless you're an expert you should use the <pre class = "all">-filtered second and minute data.
```
Comment (gaviles) [inline 03 Asset Classes/01 Equities/01 Equities/content.html]: ufff

File: 03 Asset Classes/06 Crypto/01 Crypto/content.html [hunk 13]
```diff
@@ -49,11 +49,11 @@ <h3>Live Trading</h3>
...
+		Once potential stumbling block that users face deploying their algorithms live with GDAX is misunderstanding what type of account they have with GDAX. Typically, when users first open their account with GDAX, they are given a Cash account. In other words, margin trading is not available. However, margin accounts are the default account type specified in QCAlgorithm. Therefore, when users try to place trades using methods like <div class="section-example-container"><pre>SetHoldings()</pre></div> they will get errors that crash their algorithm. Be sure to set your account type using the <div class="section-example-container"><pre>SetBrokerage()</pre></div> method.
```
Comment (jaredbroad) [inline 03 Asset Classes/06 Crypto/01 Crypto/content.html]: "<div class="section-example-container"><pre>SetHoldings()</pre></div>"

Why would this be a pre?? This should be <code> @LiXiang618

Comment (jaredbroad) [reply 03 Asset Classes/06 Crypto/01 Crypto/content.html]: And should also remove the <div class="section-example-container">

**Pull request merged**